### DISCUSSION TITLE
Can't run "brew update" from sh script but worked from terminal

### DISCUSSION THREAD

**Discussion by LeMagnifik** [Homebrew/discussions#1255] [Everyday usage]
Hello, I'm currently working on a shell script where I'll need to update homebrew and uses its commands. The issue is that all homebrew commands worked directly from terminal but didn't work from sh script.

**Comment by SMillerDev** [comment]
There's a lot of people out there who scripted their brew commands so this is unlikely to be a brew command. What does your script try and do and what is the error?

**Reply by tjluoma** [reply]
My first thought is that that `sh` script might not have the correct `$PATH` set, but until we see the script and errors, it's just guessing.
